.scaffold/tests/fixtures/init/circleci_makefile/Makefile

```
@@ -0,0 +1,93 @@
+SHELL=/bin/bash
+WEBSERVER_HOST ?= localhost
+WEBSERVER_PORT ?= 8000
+
+.PHONY: assemble, build, help, lint, lint-fix, login, provision, reset, start, status, stop, test, test-functional, test-kernel, test-unit
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)